### DISCUSSION TITLE
A command to get message replies like PnPTeamsChannelMessageReply?

### DISCUSSION THREAD

**Discussion by sbil-dev** [pnp/powershell#253] [Ideas]
Not sure if there is a PnP command to get the replies to channel messages ?

I can get all messages using PnPTeamsChannelMessage but the replies are missing.

I cant find any command like PnPTeamsChannelMessageReply in the documentation.

If there is not, then, I propose there could be a new command to get the replies. Using the graph for reply : 

https://docs.microsoft.com/en-us/graph/api/channel-get-messagereply?view=graph-rest-1.0&tabs=http

Thanks.

**Comment by Clifra-Jones** [comment]
This is really needed. Get-PnPTeamsChannelMessages is useless without the ability to get the replies. Replies should be part the data set that is returned with this function. If not then a new command to retrieve the message by the Message ID that contains the replies.
Also, there is no way that I can see to get any attachments to a message.
